consequence_prediction/repeat_expansion_variants/pipeline.py

```diff
@@ -68,37 +60,37 @@ def load_clinvar_data(clinvar_xml):
         # Append data strings
         for gene_symbol in gene_symbols:
             variant_data.append([
-                get_variant_name_or_hgvs(clinvar_record.measure),
+                clinvar_record.measure.get_variant_name_or_hgvs(),
                 clinvar_record.accession,
                 gene_symbol,
                 hgnc_id,
                 # TODO use REF and ALT to determine repeat unit length rather than just coordinate span
-                clinvar_record.measure.explicit_insertion_length
+                none_to_nan(clinvar_record.measure.coordinate_span),
+                none_to_nan(clinvar_record.measure.transcript_id),
+                none_to_nan(clinvar_record.measure.repeat_unit_length),
+                none_to_nan(clinvar_record.measure.is_protein_hgvs),
+                none_to_nan(clinvar_record.measure.repeat_type)
```

Comment:
I do not think so, three of them definitely cannot be `None`. The only one which could possibly be `None` is `clinvar_record.measure.get_variant_name_or_hgvs()` and I am not sure what would happen if that was `None` anyway.

Reply:
Actually that's a good point, I think we were only including the name/hgvs here so that it could get parsed later in the pipeline, which we no longer need since that parsing is already done by this point... maybe it's still useful to have here for debugging purposes, but you're right that it doesn't really matter whether it's `None` or `np.nan`.